packages/terra-toolkit-docs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added `useHttps` to use a secure http connection when the `gridUrl` flag is also provided.
```

Comment:
nit: this reads as if it's a source code change when this is the documentation package. We can probably rephrase this as:
```suggestion
  * Added `useHttps` flag information to `terra-functional-testing` documentation.
```

Reply:
Changed [2e82aa7](https://github.com/cerner/terra-toolkit/pull/791/commits/2e82aa7fec0992bc29e76afdf2c04951163828d8)